### ISSUE TITLE
Intro lesson doesn't find a restaurant

### ISSUE THREAD

**Issue by tobihoffmann** [osmlab/learnoverpass#47]
In the intro lesson Query doesn't find a restaurant in (around:150). Is it possible the restaurant doesn't exist anymore?
makes the follow up tasks a little weird.

**Comment by angoca** [comment]
That is true, the closer restaurant is at 450 m. I published my own fork at https://maptimebogota.github.io/learnoverpass/ that corrects this issue.

**Comment by richlv** [comment]
Duplicate of #42.
